composer/dag_test_utils/setup.py

```diff
@@ -15,13 +15,22 @@
 from setuptools import find_packages
 from setuptools import setup
 
+# read the contents of your README file
+from os import path
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+
 setup(
     name="cloud_composer_dag_test_utils",
-    version="0.0.1",
+    version="1.0.0",
     url="https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/composer/dag_test_utils",
     author="Google LLC",
     description="Utility used to unit test example Apache Airflow DAGs for Google Cloud Composer. This is not an officially supported Google product.",
+    long_description=long_description,
+    long_description_content_type='text/markdown',
     packages=find_packages(),
     py_modules=['internal_unit_testing'],
-    install_requires=['apache-airflow[google]==1.10.15']
+    install_requires=['apache-airflow[google]==2.0.0']
```

Comment:
Do you want a hard pin to 2.0.0, or should it allow versions greater or equal?

Reply:
Good question - I'm not entirely sure what best practice is here. @busunkim96 or @crwilcox do you have recommendations for installation requirements? Is it better to offer >=, a range or pin to a single version?

Reply:
Generally, you could pin a range that excludes a major version bump, unless this thing is deeply coupled to the internals of apache-airflow and can't handle even minor version mismatches.

Reply:
+1 to what Seth said. 

See [this setup.py](https://github.com/googleapis/python-bigquery/blob/27e4d34a257e2110a3928931c7ea58b2607e85d0/setup.py#L32) for an example.

If you use `==` I believe you'll have to release a new version of this package every time there's a new release of airflow.